src/components/manifold-oauth/manifold-oauth.e2e.ts

```diff
@@ -1,51 +1,84 @@
 import { newE2EPage } from '@stencil/core/testing';
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-explicit-any, no-shadow, no-param-reassign */
 
-describe('my-component', () => {
-  it('is sandboxed to only allow scripts', async () => {
-    const page = await newE2EPage();
+describe('<manifold-oauth> component', () => {
```

Comment:
I was in the process of writing a test that used `page.on('request')`, which _should_ be the Puppeteer way to monitor all network behavior. However, when used with `newE2EPage`, it just breaks all the other tests 😠.

I couldn’t think of a way to test this otherwise.